### DISCUSSION TITLE
2.6.12 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3015] [讨论]
· Mod 管理优化
　　点击安装 Mod 按钮将提示拖入 Mod 文件（#2914）
　　添加删除单个 Mod 的快捷按钮（#2921，@burningtnt）
　　添加启用、禁用单个 Mod 的快捷按钮（#2939，@SALTWOOD）
　　支持搜索 Mod 标签（#2953，@xiaoyuanjun233）
　　从 Mod 详情页返回时不会返回页面顶部（#2961，@FQO3）
　　禁用的 Mod 名称会变灰且有删除线（#2966，@FQO3）
　　增大 Mod 右下角的禁用和错误标记，使其更加明显（#2966，@FQO3）
　　兼容 .old 作为禁用后缀（#2985，@Anyplez）
　　修改下边栏禁用图标，使其与禁用标记样式一致
· 右键菜单会使用窗口透明度（#2979，@IKUN05648）
· 优化部分帮助库页面（LTCatt/PCL2Help#143，by @WForst-Breeze）
· 支持从浏览器下载窗口等更多方式拖入文件
· 由于网站访问问题，暂时移除 [历史中的今天] 主页预设
· 更新 Mod 数据库
· 更新 11 月赞助者名单，感谢大家的支持！
· 修复了以下 13 个 Bug
　　樱花披风依然没有汉化（#2845，@596wjr）
　　极个别 Mod 的默认信息加载有误（#2909，@w2w2e2wq）
　　特定操作顺序下点击前置 Mod 信息导致崩溃（#2913）
　　Snapshot 2.6.11 更新日志缺失（#2915）
　　有重复 Mod 时无法加载 Mod 信息（#2916，@xiaobaimc）
　　进入部分帮助条目时出错（#2917，@kong-que-dnf123，by @JingHai-Lingyun）
　　无法识别 Beta 版 MC 的 Mod 版本（#2931，@SuoHouDaiShi）
　　清空背景图片后出错（#2938，@LSGtorch）
　　下载 Mod 的所选详情版本可能有误（#2947，@mstower520）
　　无法安装 Forge 1.20.3~1.20.4（#2960，@WangSimele）
　　部分玩家无法打开后 [简单主页] 预设（#2963，@MCSteve123）
　　功能隐藏中更多的子页面的联动有误（#2983，@FQO3）
　　正式版修改 [使用彩色背景] 设置后不会立即刷新

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.12'>2.6.12</a>.</em>

**Comment by burningtnt** [comment]
#2921 不是我提的，我只是一个转发者

**Comment by kong-que-dnf123** [comment]
找新bug去了（bushi
